packages/medusa/src/api-v2/admin/workflows-executions/[workflow_id]/[transaction_id]/[step_id]/success/route.ts

```diff
@@ -0,0 +1,42 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { TransactionHandlerType, isDefined } from "@medusajs/utils"
+import { IWorkflowEngineService, StepResponse } from "@medusajs/workflows-sdk"
+import {
+  MedusaRequest,
+  MedusaResponse,
+} from "../../../../../../../types/routing"
+import { AdminPostWorkflowsAsyncResponseReq } from "../../../../validators"
+
+export const POST = async (req: MedusaRequest, res: MedusaResponse) => {
+  const workflowEngineService: IWorkflowEngineService = req.scope.resolve(
+    ModuleRegistrationName.WORKFLOW_ENGINE
+  )
+
+  const { workflow_id, transaction_id, step_id } = req.params
```

Comment:
thought(non-blocking): I am not super fond of creating these lengthy URLs. It makes the endpoint hard to reason about (and less importantly, not super pretty).

An alternative approach would be to include some of the params as an object:
```
POST /admin/workflow-executions/:id/steps/success
{ "transaction_id": "tx_1234", "step_id": "step_1234" }

POST /admin/workflow-executions/:id/steps/failure
{ "transaction_id": "tx_1234", "step_id": "step_1234" }
```

And the GET endpoints would be:
```
GET /admin/workflow-executions/:id

GET /admin/workflow-executions
{ "transaction_id": "tx_1234", "step_id": "step_1234" }
```

What do you think of this?

Reply:
for POST endpoints that is ok, we can add to the body, but for GET endpoints it won't change much, in the params is probably better than in the req.query.
But I don't have any strong opinion about it.

Reply:
with the params there is no typo issue that can occur whereas with query param the user need to build it and be careful of any typo, thats one of the advantage i would say